### DISCUSSION TITLE
parse response Data

### DISCUSSION THREAD

**Discussion by armanhossiny** [axmolengine/axmol#1619] [Q&A] [answered]
hi
This function does not work and adds extra characters to Json.

response->getResponseData()->data();

![10](https://github.com/axmolengine/axmol/assets/42970068/d006d1fa-f8c0-45ab-8f67-3a31468a7cab)

**Comment by halx99** [comment]
what do you mean, you should get len of data by `data.size()`

**Reply by armanhossiny** [reply]
I do the same thing, but I don't understand why that function has an extra character.

**Reply by halx99** [reply, marked as answer]
no extra characters, it's debugger try display character without null-terminated-charactor

**Reply by armanhossiny** [reply]
thank you, You're Great.
